plugin/pkg/admission/storage/persistentvolume/label/admission.go

```diff
@@ -69,7 +71,7 @@ var _ kubeapiserveradmission.WantsCloudConfig = &persistentVolumeLabel{}
 // As a side effect, the cloud provider may block invalid or non-existent volumes.
 func newPersistentVolumeLabel() *persistentVolumeLabel {
 	// DEPRECATED: cloud-controller-manager will now start NewPersistentVolumeLabelController
```

Comment:
adjust this comment to match the expected direction of using admission, since that controller depends on initializers which are being removed?